filebeat/module/nginx/access/test/test.log-expected.json

```diff
@@ -3,209 +3,216 @@
         "@timestamp": "2016-12-07T10:05:07.000Z", 
         "event.dataset": "access", 
         "event.module": "nginx", 
+        "http.request.method": "GET", 
+        "http.request.referrer": "-", 
+        "http.response.status_code": "200", 
+        "http.version": "1.1", 
         "input.type": "log", 
         "log.offset": 0, 
-        "nginx.access.body_sent.bytes": "571", 
-        "nginx.access.http_version": "1.1", 
-        "nginx.access.method": "GET", 
-        "nginx.access.referrer": "-", 
-        "nginx.access.remote_ip": "10.0.0.2", 
-        "nginx.access.remote_ip_list": [
+        "network.forwarded_ip": [
             "10.0.0.2", 
             "10.0.0.1", 
             "127.0.0.1"
         ], 
-        "nginx.access.response_code": "200", 
-        "nginx.access.url": "/ocelot", 
+        "nginx.access.body_sent.bytes": "571", 
+        "nginx.access.remote_ip": "10.0.0.2", 
```

Comment:
I assume this field should be deleted?

Reply:
It's the `keyword` field that contains either the IP or the hostname of the source.

We'd need to talk about this a bit. But essentially I haven't removed it here (or in the other access logs) yet because the visualizations built on this field would no longer display the events where a hostname was in the field, rather than an IP.